### PR TITLE
Add flow typing for pinentry component

### PR DESCRIPTION
**Pull request by awendland**
@keybase/react-hackers

### CODE REVIEW

File: shared/pinentry/index.render.js.flow [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export type Props = {
+  onSubmit: (passphrase: string, features: {[key: string]: boolean}) => void,
+  onCancel: () => void,
+  features: {[key: string]: Feature},
+  prompt: string,
+  retryLabel: string,
+  cancelLabel: ?string,
+  submitLabel: ?string,
+  windowTitle: string
```
Comment (MarcoPolo) [inline shared/pinentry/index.render.js.flow]: Looks like cancelLabel, submitLabel, and windowTitle aren't used?

Maybe we should delete those.

**Comment by MarcoPolo** [comment]
👍 % C

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

**Comment by MarcoPolo** [comment]
👍

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

**Pull request merged**